lib/modules/apostrophe-express/index.js

```diff
@@ -353,8 +353,12 @@ module.exports = {
       // Allow the options for bodyParser to be customized
       // in app.js
       var bodyParserOptions = _.extend({
-        json: { },
-        urlencoded: {extended: true}
+        json: {
```

Comment:
Is this related?

Reply:
Indeed, this is the piece that *is* related (to 1283, the issue in question). Plus per the title of the PR there's one other minor fix in the other spot.